### ISSUE TITLE
Automatically handle top-level directories?

### ISSUE THREAD

**Issue by schlich** [WetHat/MarkdownToHtml#34]
Hello, loving this product so far.  I was wondering if there is any built-in functionality toward automatically handling top-level directories in a similar manner to Mkdocs? Here is what I am looking for:

In my `markdown` folder, I have an `index.md` and a folder `reports/` with several markdown files (`report1.md`, `report2.md`, etc).  In Mkdocs, these folders are automatically handled by making the report .md files as sub-menu items of a `reports` top-level nav item.  Currently, setting my `build.json` file as so:
```
    "site_navigation": [
        { "<img width='90%' src='logo.jpg' />": "index.md" },
        { "Reports": "Reports" }
    ]
```
renders a link in the menu which takes me to a page that looks like this: 
![image](https://user-images.githubusercontent.com/21191435/156431572-db48b486-1249-4a4d-9046-a11cc85b91dc.png)

but I'm looking for a little bit more seamless configuration, whether that's a new feature that needs to be developed or something that can be done simply without manually coding a `reports.md` file and linking that to subpages, etc.  I suspect this may be possible with the current version, maybe somehow via a `build.json` config in the subdirectory, but the documentation on how sub-directories are handled are slightly thin.

Thanks in advance!

**Comment by WetHat** [comment]
Hi @schlich,
it would be conceivable to create a sub-menu structures if the link target is a directory. Currently link labels (texts) are manually configured. When we add the files of a directory how we would need to automatically generate link texts (labels) for these files. In your example, using filenames (without extension) would make perfect sense. 

To add this functionality to the current version some  processing code would need to be added to  `Build.ps1`.
That code would scan for directories in `site_navigation` and expand them. However, that would be a _one-off_. For each **new** project you would need to repeat these modifications. I could add a 'how-to' page to the documentation explaining how to modify the build file `Build.ps1` to implement such a feature. if that would be of any se to you.

To make this feature generally available a new version would be required.

**Comment by WetHat** [comment]
Fixed in [v2.7.0](https://github.com/WetHat/MarkdownToHtml/releases/tag/v2.7.0).
